package.json

```diff
@@ -63,7 +63,8 @@
     "test": "run-s test:types test:unit test:spec",
     "test:types": "tsc --noEmit",
     "test:unit": "vitest run test",
-    "test:spec": "vitest run specs"
+    "test:spec": "vitest run specs",
+    "postinstall": "nuxi prepare"
```

Comment:
Shouldn't do that. It will also run (and fail) when people install the package.

Reply:
Hmm I didn't consider that, I moved `nuxi prepare` to the `test` script now. This way the CI test job won't fail, but anyone contributing will initially be missing types for `#imports`. Should we add a `prepare` script and run it for both `test` and `dev` scripts?

Reply:
> Should we add a prepare script and run it for both test and dev scripts?

Yes, we should.

At this point, I can only come up with that idea.
The monorepo for the nuxt i18n module, unfortunately, that is root, not `packages`, so we can't use `postinstall`.
In the future, we should refactor the directory structure and make it a package structure like `docs`, `playground` and  `i18n`

Reply:
ref: nuxt-devtools: https://github.com/nuxt/devtools/blob/main/package.json

Reply:
For now I have added a `dev:prepare` script which runs before `dev` and `test`.